### ISSUE TITLE
 BiocManager::install() cannot install the package and displays an error

### ISSUE THREAD

**Issue by Wangchao-98** [Bioconductor/BiocManager#147]
This is the wrong content：
> BiocManager::install("AnnotationHub") 
Error: Bioconductor version cannot be validated; no internet connection?  See
  #troubleshooting section in vignette
In addition: Warning messages:
1: In file(con, "r") :
  URL 'https://bioconductor.org/config.yaml': status was 'SSL peer certificate or SSH remote key was not OK'
2: In file(con, "r") :
  cannot open URL 'http://bioconductor.org/config.yaml': HTTP status was '403 Forbidden'

**Comment by Wangchao-98** [comment]
This is my version of R for windows.
platform       x86_64-w64-mingw32               
arch           x86_64                           
os             mingw32                          
crt            ucrt                             
system         x86_64, mingw32                  
status                                          
major          4                                
minor          2.2                              
year           2022                             
month          10                               
day            31                               
svn rev        83211                            
language       R                                
version.string R version 4.2.2 (2022-10-31 ucrt)
nickname       Innocent and Trusting

**Comment by Wangchao-98** [comment]
I can download and install non-biocmanger installers through"install.packages()",for example "install.packages(ggplot2)"

**Comment by vjcitn** [comment]
Sorry to hear that you are having problems.  It could be that the host you are working on forbids use of the bioconductor.org domain.  Perhaps you can make use of a mirror listed at https://bioconductor.org/about/mirrors/

**Comment by vjcitn** [comment]
```
Germany
[Department of Statistics, TU Dortmund](https://www.statistik.tu-dortmund.de/)

URLs: https://bioconductor.statistik.tu-dortmund.de/; http://bioconductor.statistik.tu-dortmund.de/

Contact: Uwe Ligges <ligges at statistik.tu-dortmund.de>

[GWDG](https://ftp.gwdg.de/)

URLs: https://ftp.gwdg.de/pub/misc/bioconductor/; http://ftp.gwdg.de/pub/misc/bioconductor/

Contact: Tim Ehlers and Steffen Klemer <ftpmaster at gwdg.de>

Japan
[The Institute of Statistical Mathematics](https://www.ism.ac.jp/)

URLs: https://bioc.ism.ac.jp/; http://bioc.ism.ac.jp/

Contact: Keisuke Honda <khonda at ism.ac.jp>

[RIKEN Advanced Center for Computing and Communication](https://accc.riken.jp/en/)

URLs: https://bioconductor.riken.jp/; http://bioconductor.riken.jp/

Contact: Itoshi NIKAIDO, Ph.D. <dritoshi at gmail.com>

China
[TUNA Mirror Site, Tsinghua University](https://bioconductor.org/about/mirrors/mirrors.tuna.tsinghua.edu.cn)

URLs: https://mirrors.tuna.tsinghua.edu.cn/bioconductor/; http://mirrors.tuna.tsinghua.edu.cn/bioconductor/

Contact: Miao Wang <mirroradmin at tuna.tsinghua.edu.cn>

[eScience Center, Nanjing University](https://sci.nju.edu.cn/)

URLs: https://mirrors.nju.edu.cn/bioconductor/; http://mirrors.nju.edu.cn/bioconductor/

Contact: Ge Yao <yaoge at nju.edu.cn>

[University of Science and Technology of China (USTC)](https://ustc.edu.cn/)

URLs: https://mirrors.ustc.edu.cn/; http://mirrors.ustc.edu.cn/

Contact: Linux User Group (LUG) @ USTC <lug at ustc.edu.cn>

Norway
[University of Bergen](https://www.uib.no/)

URLs: https://bioconductor.uib.no/; http://bioconductor.uib.no/

Contact: Trond Davidsen <Trond.Davidsen at it.uib.no>

Italy
[University of Pisa](https://www.unipi.it/)

URLs: https://bioconductor.unipi.it/; http://bioconductor.unipi.it/

Contact: Fabio Pratelli <fabio.pratelli at unipi.it>

Korea
[CRAN Asia](https://cran.asia/)

URLs: https://cran.asia/; http://cran.asia/

Contact: Seongho Bae <admin at cran.asia>

Australia
[AARNet Pty Ltd](https://aarnet.edu.au/)

URLs: https://mirror.aarnet.edu.au/pub/bioconductor; http://mirror.aarnet.edu.au/pub/bioconductor

Contact: AARNet Operations Center <mirror at aarnet.edu.au>

Denmark
[dotsrc](https://dotsrc.org/)

URLs: https://mirrors.dotsrc.org/bioconductor/; http://mirrors.dotsrc.org/bioconductor/

Contact: Anders Trier Olesen <staff at dotsrc.org>

Sweden
[Academic Computer Club Umea](https://www.accum.se/)

URLs: https://mirror.accum.se/mirror/bioconductor.org/; http://mirror.accum.se/mirror/bioconductor.org/

Contact: FTP admins <ftp-adm at accum.se>
```

**Comment by Wangchao-98** [comment]
> 很抱歉听到您遇到问题。可能是您正在使用的主机禁止使用 bioconductor.org 域。也许您可以使用 https://bioconductor.org/about/mirrors/ 列出的镜子

Thank you very much for your reply. However, I was denied access to the site. Following your method I have set China (Beijing 3) [https] - Peking University in the Rsudio-tools-global options.
Unfortunately I still can't solve my problem. As far as I know, I have not found a valid solution on the Chinese CSDN site either, and I have encountered 4 peers with this same problem so far, who are also unable to solve it. I am a beginner in R. I followed the official website guide for installing BiocManger, but I could not install the R package using BiocManager::install().
I would like to ask what settings I should make for my host to finally solve this difficult problem.
I would be grateful if someone could help me！！！
